src/libs/notifications/notifications.ts

```diff
@@ -0,0 +1,105 @@
+import { Platform } from 'react-native'
+import messaging, {
+  FirebaseMessagingTypes
+} from '@react-native-firebase/messaging'
+
+import CozyClient, { generateWebLink } from 'cozy-client'
+import { navigate } from '/libs/RootNavigation'
+import { navigateToApp } from '/libs/functions/openApp'
+import { saveNotificationDeviceToken } from '/libs/client'
+
+interface NotificationData {
+  pathname: string
+  url: string
+  appName: string
+}
+
+interface InstanceOptions {
+  capabilities: {
+    flat_subdomains: string
+  }
+}
+
+export const navigateFromNotification = (
+  client: CozyClient,
+  notification: FirebaseMessagingTypes.RemoteMessage
+): void => {
+  const { data: { pathname, url, appName } = {} as NotificationData } =
```

Comment:
Can you describe why `appName` is better than `slug` here? Simple question as `slug` is a convention for cozy-apps in our team then I would like to understand the reason here.

Reply:
We should have both 
<img width="384" alt="Capture d’écran 2023-02-22 à 13 03 30" src="https://user-images.githubusercontent.com/1107936/220614701-032f2253-4423-48e1-9533-f66bcbee492b.png">

And in that case, we should use the `slug` instead of `appName`. 

`appName` is the name displayed to the user, `slug` is the technical name. So if we need to redirect someone on the right app, we have to use the `slug` and not the `appName`

https://github.com/cozy/cozy-stack/blob/63782e4e45faed57e3fa5fe0ae5c49480a5f6aaf/docs/client-app-dev.md?plain=1#L161-L162